app/models/piece.rb

```diff
@@ -31,60 +31,32 @@ def move_to!(x, y)
     end
   end
 
-# Returns true/false if a piece is obstructed in its movement.
-# Errors for movements not diagonal, vertical or horizontal
-  def is_obstructed?(x, y)
-  # Immediately return false if moving piece is a Knight, which is exempt from obstruction
+  # Returns true/false if a piece is obstructed in its movement.
+  def obstructed?(x, y)
+    # Immediately return false if moving piece is a Knight, which is exempt from obstruction
```

Comment:
I think I understand the motivation behind changing this method signature, but I also think it goes too far.

I think 'is_obstructed?' works better when applied to an explicit piece object reference given the way it reads, at least in my mind. In the Game model for example, I like to read 'unless [the] player piece [is] obstructed'...and it doesn't read that way to me any longer if the is_ portion is dropped. In the Piece model this isn't an issue since the method is never applied to an explicit reference.

An easy way to get to a middle ground would be to use **alias_method :is_obstructed?, :obstructed** in the Piece model, underneath obstructed?, and that way both signatures can be used.

Thoughts?